e2etest/config/frr/daemons

```diff
@@ -38,7 +38,7 @@ vrrpd=no
 #
 vtysh_enable=yes
 zebra_options="  -A 127.0.0.1 -s 90000000"
-bgpd_options="   -A 127.0.0.1"
+bgpd_options="   -A 127.0.0.1 -p {{.BGPPort}}"
```

Comment:
I don't think we need to set the above ?

Reply:
^^ this is to configure FRR to listen on a specified port, useful for checking the configmap's port option AND configuring 2 containers on the hostnetwork.